### ISSUE TITLE
`paraphraser` option gets selected by default

### ISSUE THREAD

**Issue by iitjee** [unijump-ai/unijump-extension#8]
<b>When I select some text </b>and click `cmd + J`, the popup window opens, with the `paraphraser` option selected. Is this intended or otherwise, the intuitive thing is to select the last opened button i.e `chat` or `paraphrase`

Appreciate your efforts in building this extension! Slick UI! 😄

**Comment by dogagenc** [comment]
@iitjee  We designed the feature intentionally. The default page is for chatting, but we assumed that if a user selects some text, they might want to paraphrase it. Do you think this feature isn't working effectively for you?

**Comment by iitjee** [comment]
@dogagenc Yes, I barely use the 'paraphraser' feature. It would be great if the `chat` option is selected, because  I can simply fetch results for  any selected text in the chatGPT without having to switch to it everytime.
